core/src/tag_utils.rs

```diff
@@ -87,6 +92,7 @@ impl SwfMovie {
 
         Ok(Self {
             header,
+            total_bytes: swf_stream.uncompressed_length,
```

Comment:
Along with the above comment, I don't think we need  a separate `total_bytes` field, it can always just return `swf_stream.uncompressed_length`.

Reply:
Sorry, abandoned it just when you started reviewing :( Give me a sec, sorry for going back and forth on this.